docs/history/changelog-4.0.rst

```diff
@@ -0,0 +1,231 @@
+.. _changelog:
+
+================
+ Change history
+================
+
```

Comment:
@thedrow the file https://github.com/celery/celery/blob/master/Changelog at the root of the repository is what you mention. I have modified it accordingly. Unless I didn't understand your comment.